code/game/objects/items/defib.dm

```diff
@@ -441,11 +439,9 @@
 		do_harm(H, user)
 		return
 
-	if((!req_defib && grab_ghost) || (req_defib && defib.grab_ghost))
+	if(can_defib(H))
 		H.notify_ghost_cloning("Your heart is being defibrillated!")
 		H.grab_ghost() // Shove them back in their body.
-	else if(can_defib(H))
-		H.notify_ghost_cloning("Your heart is being defibrillated. Re-enter your corpse if you want to be revived!", source = src)
```

Comment:
You could probably keep this message even if you grab the ghost. Would be handy if you're moving around as a ghost as you get grabbed, which would move you back out of your body by accident.

Reply:
I agree on this.

Reply:
Now a bit more forceful, grabbing the ghost right before the revival as well.